src/bases.d.ts

```diff
@@ -171,6 +171,11 @@ export interface StatefulMixin<S extends State>{
 	readonly state: S;
 
 	/**
+	 * A readonly reference to the stateFrom provided to the widget on instantiation.
+	 */
+	readonly stateFrom?: StoreObservablePatchable<S>;
```

Comment:
In implementation, wouldn't we do this as a getter, so wouldn't it be better as:

```ts
readonly stateFrom: StoreObservablePatchable<S> | undefined;
```